fractal_server/app/routes/api/v2/_aux_functions_tasks.py

```diff
@@ -211,6 +220,37 @@ async def _get_valid_user_group_id(
     return user_group_id
 
 
+async def _get_collection_status_message(
+    task_group: TaskGroupV2, db: AsyncSession
+) -> str:
+    res = await db.execute(
+        select(CollectionStateV2).where(
+            CollectionStateV2.taskgroupv2_id == task_group.id
+        )
+    )
+    state = res.scalars().all()
+    if len(state) > 1:
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=(
+                "Invalid state: expected one CollectionStateV2 associated to "
+                f"TaskGroup {task_group.id}, found {len(state)}."
+            ),
+        )
+    elif len(state) == 1 and state[0].data.get("status") in [
+        CollectionStatusV2.COLLECTING,
+        CollectionStatusV2.INSTALLING,
+        CollectionStatusV2.PENDING,
+    ]:
+        msg = (
+            f"\nTask collection is ongoing for this Task Group {task_group.id}"
```

Comment:
--> There exists a task collection (ID=) for this task group (ID=), with status {status}